libmscore/note.h

```diff
@@ -383,7 +383,7 @@ class Note final : public Element {
       void setTieFor(Tie* t)          { _tieFor = t;     }
       void setTieBack(Tie* t)         { _tieBack = t;    }
       Note* firstTiedNote() const;
-      const Note* lastTiedNote() const;
+      Note* lastTiedNote() const;
```

Comment:
This `const` should probably not be removed, it is better to add a non-const flavor of this member function:
```Note* lastTiedNote();```
It could then be used in the plugins API implementation.

Reply:
The compiler is correct, what is needed here is something like this:
```
  const Note* lastTiedNote() const { return const_cast<const Note*>(lastTiedNote()); }
  Note* lastTiedNote();
```
Note that I removed the second `const` in the second line of this example, that `const` refers not to the returned type but to the object for which this function is called, and compiler chooses which function to call depending on that `const` attribute for the type:
```
Note* n1 = new Note(score);
n1->lastTiedNote(); // Note* lastTiedNote() gets called
const Note* n2 = new Note(score);
n2->lastTiedNote(); // const Note* lastTiedNote() const gets called
```

So in order to fix the issue you should probably declare these functions like I mentioned above and remove `const` from declarations of these functions in wrappers as it doesn't seem to be needed there:
https://github.com/musescore/MuseScore/pull/5258/files#diff-3b5eed1d493c70737297ac61ccfec286R467-R468